### PR TITLE
Doc: update content

### PR DESCRIPTION
**Pull request by s-pace**
- Add faq about how many records are created
- Update the run on your own section to avoid any confusion
- Remove wrong part of the attribution policy

### CODE REVIEW

File: docs/src/faq.md [hunk 1]
```diff
@@ -118,6 +118,16 @@ encourage you to file an issue in their repository explaining how DocSearch
 could help. Feel free to [send us an email][1] as well, and we'll provide all
 the help we can.
 
+## How many records are created by DocSearch?
+
+First of all, the [parameter `nb_hits`][9] available from your configuration and
+output after every crawl is the number of records extracted from your website.
+
+The DocSearch scraper follow [the recommended atomic-reindexing strategy][10].
```
Comment (BenoitPerrot) [inline docs/src/faq.md]:
```suggestion
The DocSearch scraper follows [the recommended atomic-reindexing strategy][10].
```

File: docs/src/faq.md [hunk 2]
```diff
@@ -118,6 +118,16 @@ encourage you to file an issue in their repository explaining how DocSearch
...
+It creates a brand new temporary index to populate the data scraped from your
+website. Once the crawl is successfully achieved, this temporary index overwites
+the old index defined from your configuration with the key `index_name`.
```
Comment (BenoitPerrot) [inline docs/src/faq.md]:
```suggestion
the old index defined in your configuration with the key `index_name`.
```

File: docs/src/faq.md [hunk 3]
```diff
@@ -118,6 +118,16 @@ encourage you to file an issue in their repository explaining how DocSearch
 could help. Feel free to [send us an email][1] as well, and we'll provide all
 the help we can.
 
+## How many records are created by DocSearch?
+
+First of all, the [parameter `nb_hits`][9] available from your configuration and
```
Comment (BenoitPerrot) [inline docs/src/faq.md]:
```suggestion
The [property `nb_hits`][9] in your configuration keeps track of the number of records
```

File: docs/src/faq.md [hunk 4]
```diff
@@ -118,6 +118,16 @@ encourage you to file an issue in their repository explaining how DocSearch
...
+output after every crawl is the number of records extracted from your website.
```
Comment (BenoitPerrot) [inline docs/src/faq.md]:
```suggestion
records that were extracted and indexed by the last DocSearch run. It is updated
automatically at each run.
```

File: docs/src/run-your-own.md [hunk 5]
```diff
@@ -42,31 +42,55 @@ should check the [dedicated configuration documentation][5].
 docker run -it --env-file=.env -e "CONFIG=$(cat /path/to/your/config.json | jq -r tostring)" algolia/docsearch-scraper
 ```
 
-## Installation
+Once the crawl is done, you can [jump to the Integration step][6].
 
-The scraper is a python tool [based on scrapy][6]. We do recommend to use
+## Run the crawl from the code base
```
Comment (BenoitPerrot) [inline docs/src/run-your-own.md]:
```suggestion
## Running the crawler from the code base
```

File: docs/src/run-your-own.md [hunk 6]
```diff
@@ -42,31 +42,55 @@ should check the [dedicated configuration documentation][5].
...
 
-- [Install the driver][11] suited to your OS and the version of your Chrome. We
+- [Install the driver][12] suited to your OS and the version of your Chrome. We
   do recommend to use the latest version.
 - Set the environment variable `CHROMEDRIVER_PATH` in your `.env` file. This
   path must target the downloaded extracted driver.
 
 You are ready to go.
 
+### Run the crawl
```
Comment (BenoitPerrot) [inline docs/src/run-your-own.md]: ### Running the crawler

File: docs/src/run-your-own.md [hunk 7]
```diff
@@ -42,31 +42,55 @@ should check the [dedicated configuration documentation][5].
...
+
+You can run the crawler according to your configuration.
```
Comment (BenoitPerrot) [inline docs/src/run-your-own.md]:
```suggestion
```

File: docs/src/run-your-own.md [hunk 8]
```diff
@@ -163,12 +166,13 @@ hosted version, so you might not need all the listed commands._
...
+[6]: #integration
+[7]: https://scrapy.org/
 [8]: https://github.com/pypa/pipenv
-[9]: https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv
-[10]: ./config-file.html#js_render-optional
-[11]: http://chromedriver.chromium.org/getting-started
-[12]: https://github.com/algolia/docsearch-configs/tree/master/configs
-[13]: ./assets/playground.png
-[14]: ./dropdown.html
+[9]: https://github.com/algolia/docsearch-scraper
```
Comment (BenoitPerrot) [inline docs/src/run-your-own.md]: For the next time: _unless you are using some tool to do it for you_, please stop shifting all these references manually:
- it is unnecessary: references have an index, notably to support text reordering, there is no technical need to have them ordered by order of first appearance in the text
- it is error prone;
- it adds ~noise to the commit (see it this way: I am forced to double-check you didn't forget any reference update in the text)

Comment (s-pace) [reply docs/src/run-your-own.md]: Sure, I thought it was more clear to have it

**Pull request merged**